dlc_developer_config.toml

```diff
@@ -24,13 +24,13 @@ datetime_tag = true
 do_build = true
 
 [test]
-### On by default
 sanity_tests = true
+  safety_check_test = false
```

Comment:
nit: extra space

Reply:
this is intentional, just to show that it's an option within sanity tests